parts/linux/cloud-init/artifacts/components.json

```diff
@@ -740,7 +740,8 @@
               },
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.6.3"
+                "latestVersion": "1.6.7",
+                "previousLatestVersion": "1.6.3"
```

Comment:
This 1.6.3 is different than the container Image 1.6.5. Is this expected?

Reply:
The latestVersion was initially 1.6.3 in this field, so I put it as the previousLatestVersion here since 1.6.3 was indeed the previous latest version-- since the package section is independent of the container image section I thought this would be okay